README.md

```diff
@@ -76,7 +76,7 @@ to use the Conjur Service Broker when deploying applications, see the
     To configure the Service Broker to communicate with your external Conjur instance, the Service Broker app requires the following environment variables:
 
     - `CONJUR_VERSION`:
-      the version of your Conjur instance (`4` or `5`); defaults to 5.
+      the version of your Conjur instance (currently we only support `5`); defaults to 5.
```

Comment:
is this env var still required? if not, can we make it not required and just have it default to 5 / remove it from the docs?

we should also probably file a follow-up issue to remove it from the tile config.

Reply:
Good point - I'll remove it from the docs